src/org/labkey/test/params/FieldKey.java

```diff
@@ -0,0 +1,55 @@
+package org.labkey.test.params;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FieldKey
+{
+    private final List<String> _parts;
+    private String _label;
+
+    private FieldKey(String... parts)
+    {
+        if (parts.length == 0)
+        {
+            throw new IllegalArgumentException("No field key parts were provided.");
+        }
+        _parts = Arrays.stream(parts)
+                .map(part -> part.replace("/", "$S"))
+                .collect(Collectors.toList());
+        _label = parts[parts.length - 1];
+    }
```

Comment:
Maybe add a comment as to why you are replacing "/" with "$S"? Are you doing this because you are using INJECT_CHARS in the tests?